modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/FormBuilder.js

```diff
@@ -71,15 +71,15 @@ export function FormBuilder() {
 
 			if (groupId) {
 				globalFieldSetsPromise = getItem(
-					`/o/data-engine/v2.0/sites/${groupId}/data-definitions/by-content-type/${contentType}`
+					`/o/data-engine/v2.0/sites/${groupId}/data-definitions/by-content-type/${contentType}?page=-1&pageSize=-1`
 				);
 			}
 
 			const groupFieldSetsPromise =
 				groupId === themeDisplay.getCompanyGroupId()
 					? Promise.resolve({})
 					: getItem(
-							`/o/data-engine/v2.0/data-definitions/by-content-type/${contentType}`
+							`/o/data-engine/v2.0/data-definitions/by-content-type/${contentType}?page=-1&pageSize=-1`
```

Comment:
@IstvanD Based on this [code](https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/pagination/Pagination.java#L45), wouldn't it be enough to set only the `pageSize` (or page) to -1?

Reply:
@carolmariaabb I thought about it as well, be I figured that by setting both params it would be unambiguous that the intent is to disable pagination altogether.